openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -79,10 +80,11 @@ def create_jwt_token_dict(token_dict, oauth_adapter, use_asymmetric_key=None):
     # .. custom_attribute_name: create_jwt_grant_type
     # .. custom_attribute_description: The grant type of the newly created JWT.
     set_custom_attribute('create_jwt_grant_type', grant_type)
+    scopes = token_dict['scope'].split(' ')
```

Comment:
It feels like this should work similarly to `jwt_expires_in`, where it is processed once here and used twice below, in the JWT and the response dict.
Note: You may want to leave the old `scopes = scopes or ['email', 'profile']` as it was in `_create_jwt` just in case, but otherwise, this will make it more clear that these two values should be matching.